packages/near-membrane-dom/src/browser-realm.ts

```diff
@@ -90,22 +162,25 @@ export default function createVirtualEnvironment(
     ) {
         throw new TypeErrorCtor('Missing global object virtualization target.');
     }
+    const options = ObjectAssign(
+        {
+            __proto__: null,
+            keepAlive: false,
+        },
+        providedOptions
+    );
     const {
         distortionCallback,
         endowments,
         instrumentation,
-        keepAlive = false,
+        keepAlive,
```

Comment:
I like `keepAlive = false` more because it's an on-demand default. In other places a default make require a function call or other heavy lifting and keeping a pattern of on-demand defaults is nice.